src/overlays/effects/ovl_Effect_Ss_Stick/z_eff_ss_stick.c

```diff
@@ -8,20 +8,48 @@
 
 #define PARAMS ((EffectSsStickInitParams*)initParamsx)
 
-s32 EffectSsStick_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
+u32 EffectSsStick_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
 void EffectSsStick_Update(GlobalContext* globalCtx, u32 index, EffectSs* this);
 void EffectSsStick_Draw(GlobalContext* globalCtx, u32 index, EffectSs* this);
 
-#if 0
 const EffectSsInit Effect_Ss_Stick_InitVars = {
     EFFECT_SS_STICK,
     EffectSsStick_Init,
 };
 
-#endif
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Effect_Ss_Stick/EffectSsStick_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Effect_Ss_Stick/EffectSsStick_Draw.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Effect_Ss_Stick/EffectSsStick_Update.s")
+u32 EffectSsStick_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx) {
```

Comment:
Out of curiousity, is the `u32` needed to match? :o

Reply:
The typedef for `EffectSsInitFunc` in `z64effect.h` is `u32`, and m2c said `s32`, so I had to change one of them in order to not generate a warning, and I figured it would be here and not the typedef.

Reply:
Oh, you are right :+1: